src/shared/ui/Counter/Counter.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from "@testing-library/react";
+import userEvent from '@testing-library/user-event'
+import { act } from 'react-dom/test-utils'
+import { Counter } from "./Counter";
+
+describe("Counter Component Functionality", () => {
+    test("Should render initial value", () => {
+        const initialValue = 5;
```

Comment:
let's clarify that it's a mock data 
```suggestion
        const mockedInitialValue = 5;
```

Reply:
changed to mockedInitialValue